packages/docs/blog/2021-12-23-fhir-questionnaires.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: understanding-fhir-questionniares
+title: Understanding FHIR Questionniares
+authors:
```

Comment:
Minor nit:  you can replace this with:  `authors: reshma`

That is because `reshma` is defined in `authors.yml`.